en/drafts/originals/ocr-with-google-vision-and-tesseract.md

```diff
@@ -618,7 +624,7 @@ def new_file_layout(filename, input_dir, store_dir):
 
     lim_p[0].save(new_filepath, "PDF" ,resolution=100.0, save_all=True, append_images=lim_p[1:])
 ```
-The following function executes the above function and OCR the new PDF with the `vision_method` defined in the previous section.
+The following function executes the above and OCRs the new PDF with the `vision_method` defined [in the previous section](link).

```

Comment:
Hello @isag91. I'm not certain which section/sub-section you're referring to here? 
It would be good to add a specific link. We can use this format: `[in the previous section](#google-vision)`

Reply:
```suggestion
The following function executes the above and OCRs the new PDF with the `vision_method` defined [in the previous section](#google-vision-2).

```